docs/Documentation.md

```diff
@@ -566,15 +567,30 @@ A hook alternative to the [Progress Component](#progresscomponent).
 | bufferedPosition | `number` | The buffered position in seconds |
 | duration         | `number` | The duration in seconds          |
 
-`useTrackPlayerProgress` accepts an interval to set the rate (in miliseconds) to poll the track player's progress. The default value is `1000` or every second.
+`useTrackPlayerProgress` accepts two arguments:
+
+1. `interval`: An interval to set the rate (in miliseconds) to poll the track player's progress (default value is `1000` or every second)
+2. `pollTrackPlayerStates`: An optional whitelist of playback states at which the track player's progress is polled.
+
+  Provide `null` in case you want really poll at the `interval`'s rate, no matter the playback state the player is in.
+
+  By default the hook will only poll while a track is playing or buffering, i.e. the default value for the second argument is:
+
+  ```js
+  [
+    TrackPlayer.STATE_PLAYING,
+    TrackPlayer.STATE_BUFFERING,
+  ]
+  ```
+
```

Comment:
Can you format the parameters into a table, just like the other functions?

Reply:
I added a commit in which I added them into a table. Because the comment for the second argument is very big, I wrote a note below the table.